src/styles/main.scss

```diff
@@ -0,0 +1,12 @@
+@import "./utils/variables.scss";
```

Comment:
It is recommended to import partials in a single entry point to make it more organized. Move all imports to the beginning of the file.